dali/util/nvml.h

```diff
@@ -132,6 +137,7 @@ inline void Shutdown() {
   DALI_CALL(wrapNvmlShutdown());
 }
 
+#if (CUDART_VERSION >= 11000)
```

Comment:
Not sure if there is any value in that, but:
why not having the if inside the function and return false in case of older CUDA?

Reply:
Done